src/banknames.json

```diff
@@ -805,6 +808,7 @@
   "NILX": "Nilkanth Co-operative Bank",
   "NIRX": "Nirmal Urban Co-operative Bank Nagpur",
   "NIUX": "Nileshwar Co-operative Urban Bank",
+  "NJBK": "Nav Jeevan Co-operative Bank",
   "NJCX": "Nav Jeevan Co-operative Bank",
```

Comment:
@captn3m0 need to do a cleanup of the banknames.json once. some sublets like the above are now proper banks we have to remove all the sublet bank codes

Reply:
I don't want to remove entries from `banknames.json` unless it was added in an error. Learned this the hard way after I'd removed

>"VYSA": "ING Vysya Bank",

long back. But yeah, the sublet confusion is getting bigger, and the exceptions list is getting confusing. Need a better solution.